pyproject.toml

```diff
@@ -3,7 +3,7 @@ line-length = 100
 
 [tool.poetry]
 name = "mdai-client-py"
-version = "0"
+version = "0.8.1"
```

Comment:
Can you revert this? We specify it in `__init__.py` so it can be imported [here](https://github.com/mdai/mdai-client-py/blob/master/setup.py#L3). We currently only use poetry to lock dependencies, but it's a required field, which is why it was set to 0. Do you have any better suggestions, without needing to update versions in multiple places?

Reply:
Reverted now. I'll look into ways to avoid changing versions at different places.